### PR TITLE
Add more documentation to amuse.ic package

### PR DESCRIPTION
**Pull request by rieder**
Also make molecular_cloud and solarsystem accessible via amuse.ic

### CODE REVIEW

**Comment by rieder** [comment]
@amusecode/core please check this is OK.

File: src/amuse/__init__.py [hunk 1]
```diff
@@ -16,9 +16,9 @@
             a uniform interface to enable coupling
 
 Help is available for each of these packages, e.g.:
-> amuse.sh
->>> import amuse.ic
->>> help(amuse.ic) # doctest: +ELLIPSIS
+> ipython
+In [1]: import amuse.ic
+In [2]: help(amuse.ic) # doctest: +ELLIPSIS
```
Comment (ipelupessy) [inline src/amuse/__init__.py:21]: I don't think its necessary to use ipython for this...

Comment (rieder) [reply src/amuse/__init__.py:21]: No, but it's just an example. Python works fine too. Just wanted to get rid of amuse.sh which we shouldn't be using anymore.

Comment (rieder) [reply src/amuse/__init__.py]: fixed anyway.

**Comment by ipelupessy** [comment]
sure..

**Pull request merged**